.gitpod.yml

```diff
@@ -14,10 +14,15 @@ ports:
   - port: 9300
     onOpen: ignore
 tasks:
+  - before: >
+      nvm install 12.16.3 --reinstall-packages-from=node;
+      rvm install ruby-2.7.1
   - command: redis-server
   - command: /home/gitpod/elasticsearch-7.5.2/bin/elasticsearch
   - init: cp config/sample_application.yml config/application.yml
     command: >
+      nvm use 12.16.3;
```

Comment:
Without this the correct versions were only available in the shell session the `before` command ran in.